src/public/parts/shells/chat/public/shared/entityProfileCard.css

```
@@ -1,31 +1,19 @@
 
-.profile-popup-backdrop.show {
-	display: flex;
+.profile-popup-dialog::backdrop {
+	background: rgb(5 8 15 / 62%);
+	backdrop-filter: blur(7px);
 }
 
 .profile-popup {
-	--entity-card-accent: #5865f2;
+	--entity-card-accent: var(--color-primary);
 	position: relative;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)